ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-stable-4.15-upgrade-from-stable-4.10.yaml

```diff
@@ -0,0 +1,117 @@
+base_images:
+  ansible:
+    name: "4.15"
+    namespace: ocp
+    tag: ansible
+  cli:
+    name: "4.15"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  hypershift-operator:
+    name: "4.15"
+    namespace: ocp
+    tag: hypershift-operator
+  openstack-installer:
+    name: "4.15"
+    namespace: ocp
+    tag: openstack-installer
+  tests-private-postupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.15"
+  tests-private-preupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.14"
+  tools:
+    name: "4.15"
+    namespace: ocp
+    tag: tools
+  verification-tests:
+    name: verification-tests
+    namespace: ci
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+releases:
+  intermediate411:
+    release:
+      architecture: amd64
+      channel: candidate
+      version: "4.11"
+  intermediate412:
+    release:
+      architecture: amd64
+      channel: candidate
+      version: "4.12"
+  intermediate413:
+    release:
+      architecture: amd64
+      channel: candidate
+      version: "4.13"
+  intermediate414:
+    release:
+      architecture: amd64
+      channel: candidate
+      version: "4.14"
+  latest:
+    release:
+      architecture: amd64
+      channel: fast
```

Comment:
candidate?

Reply:
As @shellyyang1989 said, we need a stable build to setup our cluster.

Reply:
let's unify it to fast as latest version, for example in https://github.com/openshift/release/pull/46102/files#diff-e37aec18394ab645bf7b8a7bd010112149a626b9992127a9379f919d09b1177fR62